plugins/gcp/fix_plugin_gcp/resources/compute.py

```diff
@@ -6977,7 +6995,7 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
 
 
 @define(eq=False, slots=False)
-class GcpSnapshot(GcpResource):
+class GcpSnapshot(GcpResource, BaseSnapshot):
```

Comment:
snapshot_status: str = ""
    description: Optional[str] = None
    volume_id: Optional[str] = None
    volume_size: int = 0
    encrypted: bool = False
    owner_id: Optional[str] = None
    owner_alias: Optional[str] = None

Reply:
```
owner_id: Optional[str] = None
owner_alias: Optional[str] = None
```
are not accessible